services/gateway/apis/http-version3/swagger/definitions/validators.json

```diff
@@ -20,7 +24,12 @@
                         "type": "string",
                         "format": "id",
                         "example": "3c95f7931d61909ff092375fc8ad2bc35e393b62d5cca902",
-                        "description": "BLS key"
+                        "description": "BLS key."
+                    },
+                    "proofOfPossession": {
+                        "type": "string",
+                        "example": "96c5b026b1030eb73e5dfd9bfe78b0fb35e6bc7add5793fdca3d3e6a1dacb77390e998178b89f80ab8892212838bd5b2",
```

Comment:
Examples for the keys are incorrect. Can we please update the examples keys to their correct format. `generatorKey` length should be 32, `blsKey` length should be 48 and `proofOfPossession` length should be 96. What do you say @sameersubudhi ?